armi/materials/material.py

```diff
@@ -584,14 +591,15 @@ def checkTempRange(self, minT, maxT, val, label=""):
                 )
 
     def densityTimesHeatCapacity(self, Tk: float = None, Tc: float = None) -> float:
-        r"""
-        Return heat capacity * density at a temperature
+        """
+        Return heat capacity * density at a temperature.
+
         Parameters
         ----------
         Tk : float, optional
             Temperature in Kelvin.
         Tc : float, optional
-            Temperature in degrees Celsius.
+            Temperature in degrees Celsius
```

Comment:
I've got a good eye for ruff-iness now that I've reviewed a few PRs. Why was the period removed here? Just curious, obvi not a PR blocker :-)

Reply:
huh.

Typo, I guess.

Fixing it.